hr_payroll_expense/README.rst

```diff
@@ -0,0 +1,59 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Payroll Expenses Reinbursements
+===============================
+
+This module allows you to add commission and reimbursement to payslip
+
+**Features list :**
+    * Add expenses reimbursement to Payslip.
+
+**For further information:**
+    *Expenses reimbursements management: http://open-net.ch/blog/la-comptabilite-salariale-suisse-avec-odoo-1/post/salaires-avec-odoo-commissions-et-notes-de-frais-78

```

Comment:
Are you speaking about hr_payroll_expense or hr_payroll_commission ?
Because in hr_payroll_commission there is a link between hr_payslip and account_invoice and maybe that's what you are talking about.